src/core/data.jl

```diff
@@ -1,3 +1,11 @@
+"Maps component types to status parameters."
+const wm_component_status = Dict("nodes" => "status", "pipes" => "status",
+                                 "pumps" => "status", "tanks" => "status")
```

Comment:
You don't need this dict, becouse this code conforms to the `status` field standards.  PowerModels does not due to some legacy data model, but we are planning to resolve that in the next 6 months or so.